### PR TITLE
DDF for Tuya Luminance Motion Sensor Model ZG-204ZL

### PR DESCRIPTION
**Pull request by Smanar**
- Product name: Tuya Luminance Motion Sensor Model ZG-204ZL
- Manufacturer: _TZE200_3towulqd
- Model identifier: TS0601

The convertion are probably wrong, but values are here, see https://github.com/dresden-elektronik/deconz-rest-plugin/issues/6158

### CODE REVIEW

File: devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        },
+        {
+          "name": "config/tholddark"
+        },
+        {
+          "name": "config/tholdoffset"
+        },
+        {
+          "name": "state/lightlevel"
+        },
```
Comment (SwoopX) [inline devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json:128]: I guess this is misplaced here? 😄

File: devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "_TZE200_3towulqd",
+  "modelid": "TS0601",
+  "vendor": "Tuya",
+  "product": "ZG-204ZL",
```
Comment (SwoopX) [inline devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json:6]: Can you add which type of device this is, like xyz presence sensor?

**Review by SwoopX — CHANGES_REQUESTED** [review]
You also might want to add your Tuya swversion magic here on both sensors.

File: devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json (hunk 1 above)
Comment (Smanar) [reply devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json]: totaly

File: devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json (hunk 2 above)
Comment (Smanar) [reply devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json]: done

**Comment by SwoopX** [comment]
@Smanar Thanks, almost there 👍 The defaults for `config/tholddark` and `config/tholdoffset` were not necessary however, as they are already included in the respective resource item json file.

**Review by SwoopX — APPROVED** [review]

**Pull request merged**